### PR TITLE
Darwinia and Crab PLO NFT event json

### PR DESCRIPTION
**Pull request by JayJay1024**

### CODE REVIEW

File: contracts/claims/data/crab/data/nft_events.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Darwinia & Crab NFT",
+  "timestamp": "2022-08-16T00:00:00.000Z",
+  "version": { "major": 0, "minor": 1, "patch": 0 },
+  "logoURI": "https://portal.evolution.land/logos/darwinia.png",
+  "events": [
+    {
+      "chainId": 44,
+      "address": "0xB29DA7C1b1514AB342afbE6AB915252Ad3f87E4d",
```
Comment (hujw77) [inline contracts/claims/data/crab/data/nft_events.json:9]: 0xB29DA7C1b1514AB342afbE6AB915252Ad3f87E4d => 0xA561c8F6AC9eCb31e0793fBe2CCEc136a7e4bE84

The address is claims contracts ? @WoeOm

Comment (WoeOm) [reply contracts/claims/data/crab/data/nft_events.json:9]: Should be the claims contract address

**Review by hujw77 — APPROVED** [review]

File: contracts/claims/data/crab/data/nft_events.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "timestamp": "2022-08-16T00:00:00.000Z",
+  "version": { "major": 0, "minor": 1, "patch": 0 },
+  "logoURI": "https://portal.evolution.land/logos/darwinia.png",
+  "events": [
+    {
+      "chainId": 44,
+      "address": "0xA561c8F6AC9eCb31e0793fBe2CCEc136a7e4bE84",
+      "name": "3nd Wave",
+      "detail":"-",
+      "logoURI": "https://portal.evolution.land/logo192.png",
```
Comment (WoeOm) [inline contracts/claims/data/crab/data/nft_events.json:12]: use Darwinia Logo

Comment (JayJay1024) [reply contracts/claims/data/crab/data/nft_events.json]: resolved

**Review by WoeOm — APPROVED** [review]

**Comment by WoeOm** [comment]
LGTM

**Pull request merged**